src/main/java/hudson/tasks/test/TestResultProjectAction.java

```diff
@@ -105,16 +105,16 @@ public String getUrlName() {
     }
 
     public AbstractTestResultAction getLastTestResultAction() {
-        final Run<?,?> tb = job.getLastSuccessfulBuild();
+        //final Run<?,?> tb = job.getLastSuccessfulBuild();
```

Comment:
why remove this?

Reply:
Why should only be the last successful build a build with test results? I think any test results are test results, regardless of the build failure. Nowadays pipeline builds can be failed, even though just a substage failed, not the whole build. Using UNSTABLE is not feasible for this, as that does not mark a build as containing a failure for other systems that list the Jenkins builds externally.

Reply:
true, can you remove the commented out code please